### DISCUSSION TITLE
Managing calcs with categories or tags

### DISCUSSION THREAD

**Discussion by soundstep** [kazad/instacalc#63] [Ideas]
First off, I love this app!
It would be really great to be able to sort or categorize the calcs when you have a lot of them! Any kind of tree, folder structure, label, or tag system would do! That's it!
Cheers.

**Comment by kazad** [comment]
Glad you like it! Thanks for the suggestion, this has been on the backlog for a bit. I'll put this into the release cycle :).

**Comment by kazad** [comment]
Got this one out! Should be live now, see https://github.com/kazad/instacalc/discussions/64
